.github/workflows/updateProtectedBranch.yml

```diff
@@ -1,140 +0,0 @@
-# This is a utility workflow to create, approve, and merge a pull request against a protected branch.
-name: Update Protected Branch
-
-on:
-  workflow_dispatch:
-    inputs:
-      TARGET_BRANCH:
-        description: The target branch to update. This becomes the base branch of the pull request.
-        required: true
-      SOURCE_BRANCH:
-        description: If updating main, you must also provide a head branch to update main with.
-        required: false
-        default: ''
-
-jobs:
-  updateBranch:
-    if: github.actor == 'OSBotify'
```

Comment:
Reviewer note: This check is not needed anymore. Because it's now an action and not a manually-triggerable workflow, there's no way a non-bot could trigger this action manually, which is why we had this check in the first place.